packages/services/f-feature-management/src/vue/services/features.service.js

```diff
@@ -0,0 +1,103 @@
+import createFeatureManagementInstance from '../../lib/createFeatureManagementInstance';
+import featuresModule from '../store/features.module';
+
+const namespace = 'f-feature-management';
+const storeAction = `${namespace}/update`;
+
+function trackExperiment (key, variant, analytics) {
+    analytics.pushEvent({
+        event: 'trackExperimentV3',
+        experiment: {
+            key,
+            variant,
+            platform: 'feature_management',
+            version: 1,
+            deviceTimestamp: new Date().toISOString()
+        }
+    });
+}
+
+function contextGetter (store, cookies) {
+    return {
+        anonUserId: cookies.get('je-auser') || null,
+        country: store.state.globalisation.tenant
+    };
+}
+
+function createSettings (store, serviceStore, cookies, analytics, logger) {
+    const { featureManagement: envConfig } = store.state.configuration.settings;
+
+    const settings = {
+        keyPrefix: envConfig.scope,
+        cdn: {
+            host: envConfig.host,
+            scope: envConfig.scope,
+            environment: envConfig.environment,
+            key: envConfig.obfuscationKey,
+            poll: (envConfig.pollInterval > 0),
+            pollInterval: envConfig.pollInterval
+        },
+        contextGetter: () => contextGetter(store, cookies),
+        onTrack: (key, variant) => trackExperiment(key, variant, analytics)
+    };
+
+    if (logger) {
+        settings.logger = logger;
+    }
+
+    const initialConfigAsJson = serviceStore && serviceStore.configJson;
+    if (initialConfigAsJson && initialConfigAsJson.features) {
+        logger.logInfo('[FeaturesService] Initialising with stored config');
+        settings.initialConfigAsJson = initialConfigAsJson;
+    }
+
+    return settings;
+}
+
+function createLoggerWrapper (store, logger) {
+    const tags = ['feature-management'];
+    return {
+        logInfo: message => logger && logger.logInfo(message, store, { tags }),
+        logWarn: message => logger && logger.logWarn(message, store, { tags }),
+        logError: message => logger && logger.logError(message, store, { tags })
```

Comment:
Should be able to use optional chaining here

```suggestion
        logInfo: message => logger?.logInfo(message, store, { tags }),
        logWarn: message => logger?.logWarn(message, store, { tags }),
        logError: message => logger?.logError(message, store, { tags })
```